cleo/commands/completions_command.py

```diff
@@ -120,266 +120,148 @@ def handle(self) -> int:
         return 0
 
     def render(self, shell: str) -> str:
-        return getattr(self, f"render_{shell}")()
-
-    def render_bash(self) -> str:
-        template = TEMPLATES["bash"]
+        return getattr(self, f"render_{shell}")()  # type: ignore[no-any-return]
 
+    def _get_script_name_and_path(self) -> tuple[str, str]:
+        # TODO: add support for generating via `python -m cli`
```

Comment:
what exactly do you mean by this?

Reply:
yea I'll try to reword this for clarity, but I meant that our `script_name` inference doesn't exactly work when running `python3 -m poetry completions zsh`. In that case, we end up thinking the script is called `__main__.py` rather than `poetry`.

This was something I noticed that Arch Linux has to workaround: https://github.com/archlinux/svntogit-community/blob/packages/python-poetry/trunk/poetry-completions-generator

Reply:
Could you make an issue out of this? Just so we don't forget (seems quite significant).

Reply:
Agreed, made #231